Source/ui_unix/settingsdialog.cpp

```diff
@@ -0,0 +1,88 @@
+#include "settingsdialog.h"
+#include "ui_settingsdialog.h"
+
+#define PREFERENCE_UI_SHOWFPS "ui.showfps"
+#define PREFERENCE_UI_SHOWVIRTUALPAD "ui.showvirtualpad"
+#define PREFERENCE_AUDIO_ENABLEOUTPUT "audio.enableoutput"
+#define PREF_CGSH_OPENGL_ENABLEHIGHRESMODE        "renderer.opengl.enablehighresmode"
```

Comment:
Preference defines for the OpenGL GS handler and the base GL handler will be available if you include `../gs/GSH_OpenGL/GSH_OpenGL.h". No need to redefine them here.